packages/paste-core/components/chat-composer/src/ChatComposerAttachment.tsx

```diff
@@ -0,0 +1,49 @@
+import type { BoxElementProps } from "@twilio-paste/box";
+import { Box } from "@twilio-paste/box";
+import { MediaBody, MediaFigure, MediaObject } from "@twilio-paste/media-object";
+import { Stack } from "@twilio-paste/stack";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import * as React from "react";
+
+export interface ChatComposerAttachmentProps extends HTMLPasteProps<"div"> {
+  children: NonNullable<React.ReactNode>;
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   *
+   * @default "CHAT_COMPOSER_ATTACHMENT"
+   * @type {BoxProps["element"]}
+   * @memberof ChatComposerAttachmentProps
+   */
+  element?: BoxElementProps["element"];
+  /**
+   * Pass an icon to use for the attachment message. DownloadIcon recommended
+   *
+   * @default null
+   * @type {NonNullable<React.ReactNode>}
+   * @memberof ChatComposerAttachmentProps
+   */
+  attachmentIcon: NonNullable<React.ReactNode>;
+}
+
+const ChatComposerAttachment = React.forwardRef<HTMLDivElement, ChatComposerAttachmentProps>(
```

Comment:
suggestion: can this component be combined with ChatComposerAttachmentCard to reduce API complexity?

Reply:
Absolutely!